src/frontend/src/lib.rs

```diff
@@ -159,6 +160,10 @@ pub struct FrontendOpts {
         default_value = "./secrets"
     )]
     pub temp_secret_file_dir: String,
+
+    /// Total available memory for the frontend node in bytes. Used by both computing and storage.
```

Comment:
Does storage run in frontend? IIRC, even in local execution mode, the `TableScan` is pushed to compute nodes.